src/components/form/style.css

```diff
@@ -2,11 +2,13 @@ smoothly-form {
 	display: block;
 	--background-color: var(--smoothly-default-color);
 }
-smoothly-form > form {
+
+smoothly-form>form {
```

Comment:
why have all the selectors changed format from `element > element` to `element>element` ?